docs/_publish/launch-process/zps.md

```diff
@@ -0,0 +1,113 @@
+---
+title: Zapier Partner Sandbox
+order: 4
+layout: post-toc
+---
+
+# Zapier Partner Sandbox
+
+## What is Zapier Partner Sandbox?
+Zapier Partner Sandbox is a workspace for people in your organization who are on your integration team. It offers qualifying partners several premium features for free. Zapier Partner Sandbox is designed to support the ongoing development of your integration. Features of Zapier Partner Sandbox include:
+
+- Unlimited Zaps
+- 2,000 free tasks per month
+    - Actions for Zapier apps do not count towards task usage
+- 5-minute update time for polling Zaps
+- Multi-step Zaps
+- 5 Premium Integrations
+- Paths (3 paths wide by 3 paths deep)
+
+Zapier Partner Sandbox does not include premium features for Zapier Tables or Zapier Interfaces.
+
+## Who’s eligible for Zapier Partner Sandbox?
+Any Admin or Collaborator on a public integration team can request access to Zapier Partner Sandbox.
```

Comment:
to confirm, this is unlimited? so they can have multiple admins on this plan? if so, I'd suggest rewording to:

Each Admin or Collaborator on a public integration team can request access to Zapier Partner Sandbox.